### PR TITLE
README translated to pt-br

### PR DESCRIPTION
**Pull request by douglas-DS**
Here is my contribution for your great project

### CODE REVIEW

File: .github/README_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```
Comment (shpotainna) [inline .github/README_pt-br.md]: Just a small comment. Please, change here `25px` to `20px` as it isn't English translation. And for Brazilian flag : `20px` to `25px`.

```suggestion
        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```

File: .github/README_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/ru.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_zh.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/cn.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_ko.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/kr.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_pt-br.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/br.svg">
```
Comment (shpotainna) [inline .github/README_pt-br.md]:
```suggestion
        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/br.svg">
```

File: .github/README_pt-br.md (hunk 1 above)
Comment (douglas-DS) [reply .github/README_pt-br.md]: My bad, i'm completely forgot to change flags size  when it is the currently language

**Comment by Shpota** [comment]
Thank you very much!

**Pull request merged**